README.md

```diff
@@ -29,7 +29,7 @@ There are several ways to get started using .NET with Jupyter, including Jupyter
 
 ## Visual Studio Code
 
-Work is underway to add support for the new Visual Studio Code native notebook feature. While we are still in the early stages of this effort, if you'd like to experiment with it you can find instructions [here](../src/dotnet-interactive-vscode/README.md). 
+Work is underway to add support for the new Visual Studio Code native notebook feature. While we are still in the early stages of this effort, if you'd like to experiment with it you can find instructions [here](./src/dotnet-interactive-vscode/README.md). 
```

Comment:
Should we like to current VS Code Native NB extension?

Reply:
I added a link.